include/swift/AST/AnyFunctionRef.h

```diff
@@ -151,7 +151,10 @@ class AnyFunctionRef {
     }
     llvm_unreachable("unexpected AnyFunctionRef representation");
   }
-  
+
+// Disable "only for use within the debugger" warning
+#pragma warning(push)
```

Comment:
*Clang* still has unknown pragma warnings on, so this should be guarded by a check for MSVC.